go/pkg/client/cas_test.go

```diff
@@ -616,18 +621,12 @@ func TestUploadConcurrentCancel(t *testing.T) {
 			fake.ReqSleepDuration = reqMaxSleepDuration
 			fake.ReqSleepRandomize = true
 			c := e.Client.GrpcClient
-			client.CASConcurrency(defaultCASConcurrency).Apply(c)
```

Comment:
It looks like this was completely removed-  is this intentional? I guess it is since I see that `tc.concurrency` is being applied right after.

Reply:
Yep, idk why I had it here to begin with.